src/main/scala/io/qbeast/spark/utils/QbeastExpressionUtils.scala

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2021 Qbeast Analytics, S.L.
+ */
+package io.qbeast.spark.utils
+
+import io.qbeast.core.model._
+import io.qbeast.spark.internal.expressions.QbeastMurmur3Hash
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.sql.catalyst.expressions.{
+  And,
+  BinaryComparison,
+  EqualTo,
+  Expression,
+  GreaterThanOrEqual,
+  LessThan,
+  Literal,
+  SubqueryExpression
+}
+import org.apache.spark.sql.types._
+
+/**
+ * Object with utility methods to work with Spark expressions.
+ */
+object QbeastExpressionUtils {
```

Comment:
What's the goal of this utility class? If this class converts the SQL predicate to QuerySpace, we should use a more meaningful name. Also, if it is an internal class, better to limit its visibility.

Reply:
Moved to QuerySpec class on ae8b2c829e180b48870f0fc62ef32d3cf5769dfe